### ISSUE TITLE
Option to use an organization instead.

### ISSUE THREAD

**Issue by noirscape** [kashav/point#3]
Currently the tool uses my own repositories. This has the disatvantage of cluttering the public repository list with 'shortened' urls. This problem could be migitated if it were possible to use an organization to store the shortened urls in instead.

**Comment by kashav** [comment]
I agree that this is a problem, I believe #1 also helps to solve this.

I like the idea, especially for organizations. Going to have to do some quick research about organization push/pull access from individual accounts through the API, but I don't think that'll be a problem.

Interested in hearing more about how an individual account would use this though, would you use a dedicated GitHub organization to store shortened URLs?

This might also call for some sync feature, since presumably multiple people will be pushing to the same organization.

Would love to hear any thoughts you have about this.

**Comment by noirscape** [comment]
@kshvmdn Yep. Basically I'd put shortened repositories into a serparate organization (let's call it foo) that the program then uses instead of my own user-profile, creating repositories and urls like 'http://foo.github.io/bar' instead of 'http://ev1l0rd.github.io/kFG2g8rr'. 

That would have the advantage of allowing the shortened URLs not be the users name and of course the aforementioned clutter.

A sync feature could be implemented by creating some sort of meta repository in which a config file is stored that contains the current shortened url list. When adding/deleting a repository, the program could then pull that file (no real need to keep histories here), add/delete the relevant entry and then remove the meta repo and recreate it with the meta file inside it. Not sure on how hard that would be to implement, but that could be a concept.
